web/src/locales/de/data.json

```diff
@@ -29,6 +29,7 @@
     "ZH 🡲 EN": "ZH 🡲 EN"
   },
   "chat": {
+    " Release the uploaded file": " Release the uploaded file",
```

Comment:
See: https://casdoor.org/docs/contributing/#how-to-update-i18n-data

Zh needs transaltion